2024-03-25 17:48:32,646 INFO werkzeug [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2024-03-25 17:48:32,646 INFO werkzeug [33mPress CTRL+C to quit[0m
2024-03-25 17:48:56,262 INFO root Starting home function
2024-03-25 17:48:56,347 INFO werkzeug 127.0.0.1 - - [25/Mar/2024 17:48:56] "GET / HTTP/1.1" 200 -
2024-03-25 17:48:56,532 INFO werkzeug 127.0.0.1 - - [25/Mar/2024 17:48:56] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
2024-03-25 17:48:56,565 INFO werkzeug 127.0.0.1 - - [25/Mar/2024 17:48:56] "[36mGET /static/phocacssflagswidth/css/phoca-flags.css HTTP/1.1[0m" 304 -
2024-03-25 17:48:56,573 INFO werkzeug 127.0.0.1 - - [25/Mar/2024 17:48:56] "[36mGET /static/phocacssflagswidth/css/style.css HTTP/1.1[0m" 304 -
2024-03-25 17:48:56,578 INFO werkzeug 127.0.0.1 - - [25/Mar/2024 17:48:56] "[36mGET /static/img/logo.svg HTTP/1.1[0m" 304 -
2024-03-25 17:49:41,721 INFO root Starting add function
2024-03-25 17:49:41,725 INFO root Finished deleting jobs
2024-03-25 17:49:41,728 INFO root Finished retrieving user request
2024-03-25 17:49:41,729 ERROR app Exception on /add [POST]
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\webscrapers\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\Anaconda3\envs\webscrapers\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\Anaconda3\envs\webscrapers\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\Anaconda3\envs\webscrapers\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Nadia Stoyanova\Documents\GitHub\Web-scraping-jobs\app\app.py", line 187, in add
    with open(json_test, "w") as outfile:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Nadia Stoyanova\\Documents\\GitHub\\data\\jobs_parameters_user_request.json'
2024-03-25 17:49:41,731 INFO werkzeug 127.0.0.1 - - [25/Mar/2024 17:49:41] "[35m[1mPOST /add HTTP/1.1[0m" 500 -
